packages/dds/task-manager/src/interfaces.ts

```diff
@@ -47,6 +49,32 @@ export interface ITaskManager extends ISharedObject<ITaskManagerEvents> {
     queued(taskId: string): boolean;
 }
 
+export interface IOldestClientObservableEvents extends IEvent {
```

Comment:
Interface name does not map signature. I.e. there is nothing about "oldest client" here, no?
Also comment below seems out of place (it feels like it refers to some place where you need to cast IFluidDataStoreRuntime to this interface).

Reply:
This follows existing conventions in the repo for naming the event generic for `IEventProvider<T>` based on the event provider interface name:
```typescript
interface IFoobarEvents extends IEvent { ... }
interface IFoobar extends IEventProvider<IFoobarEvents> { ... }
```

Re: the comment, I wanted to explain why I've added the eslint exception, as opposed to:
```typescript
(event: "connected" | "disconnected", listener: () => void);
```

`ContainerRuntime` and `IFluidDataStoreRuntime` both emit `"connected"` with a `clientId: string` arg, (which is OK to omit as I've done here), but Typescript seems unhappy if I omit that arg from a unified event signature.